### DISCUSSION TITLE
[@fluentui/react-charting] Grouped Vertical Bar Chart - Bar overlaps when in single series

### DISCUSSION THREAD

**Discussion by jekcalingan** [microsoft/fluentui#29326] [General] [answered]
Hi team,

I noticed that the bar overlaps when I only have a single series. Any idea how I can resolve this issue? Thanks.

@fluentui/react-charting v5.17.7

Single Series:
![image](https://github.com/microsoft/fluentui/assets/65863251/e8ee7b3e-3be7-471c-9ac5-e7b8d647eaf8)

Multi Series
![image](https://github.com/microsoft/fluentui/assets/65863251/d319af64-77c2-4a05-8ec8-d6457a42a884)

Edited:
I see that the width is being set to a negative value based on console log error.
![image](https://github.com/microsoft/fluentui/assets/65863251/d8c7cf55-1583-4ac3-8753-96283d4e8304)

**Comment by AtishayMsft** [comment, marked as answer]
Hey @jekcalingan. This bug seems to have been introduced in version 5.16.0 of the library.
Could you try installing the version 5.15.10 where this issue does not happen.
We will plan to fix this issue soon.

**Reply by jekcalingan** [reply]
Thanks @AtishayMsft, this workaround is working. However, how do we know if the fix for this is already released?

**Comment by AtishayMsft** [comment]
We have a bug tracking this issue. I also created an issue #29335 in this repo that you can track for updates and resolution.

**Reply by jekcalingan** [reply]
Alright, thank you very much for your help.

**Reply by AtishayMsft** [reply]
@jekcalingan This issue has been resolved in [5.18.0](https://github.com/microsoft/fluentui/tree/@fluentui/react-charting_v5.18.0) of the react-charting library.

**Reply by jekcalingan** [reply]
@AtishayMsft Alright, will check it out, thanks for your help.
